youtbestyle2.css

```diff
@@ -4,7 +4,7 @@
     display: flex;
     flex-direction: row;
     align-items: center;
-    /* justify-content: space-between; */
```

Comment:
**[Optional]**

- [ ] It would be nice if you add some emojis in your inline comments to make them human friendly 😊